client/js/recurring_donations/edit/index.es6

```diff
@@ -199,9 +198,9 @@ function view(state) {
         h('td.strong', 'Address')
       , h('td', [
           h('small', [
-            [supporter.address, supporter.city].filter(R.identity).join(', ')
+            [supporter.address, supporter.city].filter(i => i).join(', ')
```

Comment:
Is this filter here to strip out elements that are empty?

Reply:
But I can try to investigate further what happens if one is omitted